optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -159,7 +165,7 @@ export const GraphHistory: FC<{
               control={
                 <Checkbox
                   checked={!filterPrunedTrial}
-                  disabled={!study?.has_intermediate_values}
+                  disabled={!studies[0]?.has_intermediate_values}
```

Comment:
Could you avoid using `!` to unwrap `StudyDetail | null`? It may crash the entire application.

Reply:
Sorry if I understand your question wrongly, but this `!` is for logical operator, not for `non-null assertion operator`.

Reply:
Oh, sorry. You are correct. I misunderstood the code. Let me resolve this.